lib/swagger_client/models/employment_summary_container_v3_0.rb

```diff
@@ -0,0 +1,175 @@
+require 'date'
+
+module SwaggerClient
+  class EmploymentSummaryContainerV30
+    attr_accessor :employment_summary
+
+    # Attribute mapping from ruby-style variable name to JSON key.
+    def self.attribute_map
+      {
+        :'employment_summary' => :'employment-summary'
+      }
+    end
+
+    # Attribute type mapping.
+    def self.swagger_types
+      {
+        :'employment_summary' => :'EmploymentSummaryV30'
+      }
+    end
+
+    # Initializes the object
+    # @param [Hash] attributes Model attributes in the form of hash
+    def initialize(attributes = {})
+      return unless attributes.is_a?(Hash)
+
+      # convert string to symbol for hash key
+      attributes = attributes.each_with_object({}) { |(k, v), h| h[k.to_sym] = v }
+
+      if attributes.has_key?(:'employment-summary')
+        self.employment_summary = attributes[:'employment-summary']
+      end
+    end
+
+    # Show invalid properties with the reasons. Usually used together with valid?
+    # @return Array for valid properties with the reasons
+    def list_invalid_properties
+      invalid_properties = Array.new
+      invalid_properties
+    end
+
+    # Check to see if the all the properties in the model are valid
+    # @return true if the model is valid
+    def valid?
+      true
+    end
+
+    # Checks equality by comparing each attribute.
+    # @param [Object] Object to be compared
+    def ==(o)
+      return true if self.equal?(o)
```

Comment:
this method seems like it needs a bit of cleanup since the fields referenced aren't on this model.